src/main/java/ch/epfl/biop/imaris/demo/GetSurfacesLabelDemo.java

```diff
@@ -0,0 +1,56 @@
+package ch.epfl.biop.imaris.demo;
+
+import Imaris.IApplicationPrx;
+import Imaris.IDataSetPrx;
+import Imaris.ISurfacesPrx;
+import ch.epfl.biop.imaris.EasyXT;
+import ch.epfl.biop.imaris.ImarisCalibration;
+import ch.epfl.biop.imaris.SurfacesDetector;
+import net.imagej.ImageJ;
+import ij.*;
+
+public class GetSurfacesLabelDemo {
+
+    public static void main(String... args) throws Imaris.Error {
+        ImageJ ij = new ImageJ();
+        ij.ui().showUI();
+
+        // Fresh Start with the sample dataset
+        FreshStartWithIJAndBIOPImsSample.main();
+
+        // get Imaris App if you need to use Imaris API
+        IApplicationPrx imaris_app = EasyXT.getImaris();
+        // eg to get the dataset we can use and here the dataset with calibration
+        IDataSetPrx dataset = imaris_app.GetDataSet();
```

Comment:
Is this for demo? there is a EasyXT.getCurrentDataSet() also

Reply:
7285d0764182e42609b782f1c122f3d37d9d8ed4